bin/oref0-log-shortcuts.sh

```diff
@@ -1,13 +1,23 @@
 #!/bin/bash
 
 # add crontab entries
-grep -q networklog ~/.bash_profile 2>/dev/null || echo "alias networklog="'"tail -n 100 -F /var/log/openaps/network.log"' >> ~/.bash_profile
+grep -q network-looplog ~/.bash_profile 2>/dev/null || echo "alias network-looplog="'"tail -n 100 -F /var/log/openaps/network.log"' >> ~/.bash_profile
 grep -q xdrip-looplog ~/.bash_profile || echo "alias xdrip-looplog="'"tail -n 100 -F /var/log/openaps/xdrip-loop.log"' >> ~/.bash_profile
 grep -q cgm-looplog ~/.bash_profile || echo "alias cgm-looplog="'"tail -n 100 -F /var/log/openaps/cgm-loop.log"' >> ~/.bash_profile
 grep -q autosens-looplog ~/.bash_profile || echo "alias autosens-looplog="'"tail -n 100 -F /var/log/openaps/autosens-loop.log"' >> ~/.bash_profile
-grep -q autotunelog ~/.bash_profile || echo "alias autotunelog="'"tail -n 100 -F /var/log/openaps/autotune.log"' >> ~/.bash_profile
+grep -q autotune-looplog ~/.bash_profile || echo "alias autotune-looplog="'"tail -n 100 -F /var/log/openaps/autotune.log"' >> ~/.bash_profile
```

Comment:
I don't think we want to add "loop" to these, as that's not present in the name of the logfile that's being tailed.